tests/manage/pv_services/test_ceph_capacity_recovery.py

```diff
@@ -0,0 +1,239 @@
+import logging
+import time
+
+from ocs_ci.ocs.perftests import PASTest
+
+from ocs_ci.ocs import node, constants
+from ocs_ci.helpers import helpers
+from ocs_ci.ocs.exceptions import (
+    PodNotCreated,
+    CephHealthException,
+    TimeoutExpiredError,
+)
+from ocs_ci.helpers.managed_services import (
+    get_used_capacity,
+    verify_osd_used_capacity_greater_than_expected,
+)
+
+from ocs_ci.ocs.resources import pvc, ocs
+from ocs_ci.utility.utils import TimeoutSampler, ceph_health_check_base
+
+from ocs_ci.ocs.benchmark_operator_fio import get_file_size, BenchmarkOperatorFIO
+
+log = logging.getLogger(__name__)
+
+
+def check_health_status():
+    """
+    Exec `ceph health` cmd on tools pod to determine health of cluster and logs the results
+
+    Returns:
+        boolean: True if HEALTH_OK
+
+    """
+    try:
+        status = ceph_health_check_base()
+        if status:
+            log.info("Health check passed")
+        else:
+            log.info("Health check failed")
+        return status
+    except CephHealthException as e:
+        # skip because ceph is not in good health
+        log.info(f"Ceph health exception received: {e}")
+        return False
+
+
+class TestCephCapacityRecovery(PASTest):
```

Comment:
Please add skipif marker if the feature is available in a particular verision.

Reply:
The feature is available after fix of https://bugzilla.redhat.com/show_bug.cgi?id=1910272 ( fix in ceph). 
I 've tested it on ODF 4.12 and it worked. Adding a skipif marker for version < 4.12.